### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by wan92hen**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
增加部署至阿里云图标

```release-note
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +19,9 @@
 <a href="https://gitee.com/halo-dev">Gitee</a>
 <a href="https://t.me/halo_dev">Telegram 频道</a>
 </p>
+<p align="center">
+    <a href="https://market.aliyun.com/common/dashi/halo?userCode=kmemb8jp" target="_blank"><img src="https://img.alicdn.com/imgextra/i2/O1CN01H5JIwY1rZ0OobDjnJ_!!6000000005644-2-tps-1000-216.png" alt="1Panel-dev%2FMaxKB | Aliyun" style="width: 250px; height: auto;" /></a>
```
Comment (ruibaby) [inline README.md:23]:
```suggestion
    <a href="https://market.aliyun.com/common/dashi/halo?userCode=kmemb8jp" target="_blank"><img src="https://img.alicdn.com/imgextra/i2/O1CN01H5JIwY1rZ0OobDjnJ_!!6000000005644-2-tps-1000-216.png" style="width: 250px; height: auto;" /></a>
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Review by ruibaby — APPROVED** [review]
/lgtm

**Pull request merged**